helm/ffc-demo-web/templates/ingress-alb.yaml

```diff
@@ -2,25 +2,25 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ .Values.name }}
+  name: {{ quote .Values.name }}
   annotations:
     kubernetes.io/ingress.class: alb
     alb.ingress.kubernetes.io/scheme: internal
-    alb.ingress.kubernetes.io/security-groups: {{ .Values.ingress.alb.securityGroups}}
-    alb.ingress.kubernetes.io/certificate-arn: {{ .Values.ingress.alb.arn}}
-    alb.ingress.kubernetes.io/tags: {{ .Values.ingress.alb.tags }}
+    alb.ingress.kubernetes.io/security-groups: {{ quote .Values.ingress.alb.securityGroups}}
+    alb.ingress.kubernetes.io/certificate-arn: {{ quote .Values.ingress.alb.arn}}
+    alb.ingress.kubernetes.io/tags: {{ quote .Values.ingress.alb.tags }}
     external-dns.alpha.kubernetes.io/aws-zone-type-private: "true"
 spec:
   rules:
 {{- if .Values.ingress.endpoint }}
-  - host: {{ .Values.ingress.endpoint }}.{{.Values.ingress.server}}
+  - host: {{ quote .Values.ingress.endpoint }}.{{quote .Values.ingress.server}}
```

Comment:
I suspect this line is invalid as it puts quotes within the ingress endpoint value as well as around it, so it would end up as `ffc-demo"."vividcloudsolutions.co.uk`